files/build_templates/organization_extensions.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+#########################################################################
+## This script is to automate Orignaization specific extensions 	#
+## such as Configuration & Scripts for features like AAA, ZTP, etc.	#
+## to include in ONIE installer image					#
+##									#
+## USAGE:								#
+##   ./organization_extensions.sh FILESYSTEM_ROOT HOSTNAME		#
```

Comment:
can you use getopt to add options, this will allows the others to add their options without causing backward compatibility issue.